distribution/tools/keystore-cli/src/test/java/org/elasticsearch/cli/keystore/ListKeyStoreCommandTests.java

```diff
@@ -41,23 +45,23 @@ public void testEmpty() throws Exception {
         createKeystore(password);
         terminal.addSecretInput(password);
         execute();
-        assertEquals("keystore.seed\n", terminal.getOutput());
+        assertThat(terminal.getOutput().lines().toList(), contains("keystore.seed"));
```

Comment:
No need to parse it using `lines()`, just use `containsString`?

Reply:
My thinking here was that `lines()` gives us a list containing a string which must be exactly `keystore.seed`, which is a stronger check than whether whether the output contains a substring.